.github/workflows/build.yml

```diff
@@ -6,7 +6,9 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: cypress/browsers:node14.17.0-chrome91-ff89
+    container:
+      image: cypress/browsers:node16.16.0-chrome105-ff104-edge
+      options: --user 1001
```

Comment:
Why do we need to add this `options: --user 1001` parameter? I didn't find anything related to this in the cypress documentation 😅

Reply:
Nvm, I've found it! In case anyone also wonders why, [read](https://github.com/cypress-io/github-action#firefox).

Reply:
I still don't understand why it is necessary. That's the only reference I found in their docs, asides from some issues[ like this one](https://github.com/cypress-io/cypress-docker-images/issues/560). We are not using Firefox yet, but seems that the image lacks certain permissions when installing?

[This is the error](https://github.com/empathyco/x/actions/runs/3131223359/jobs/5082443562) that gave without that user.
![image](https://user-images.githubusercontent.com/10955146/193896892-a6f6beae-cd78-4e05-ad87-43d297f134af.png)